frameworks/elastic/src/main/dist/elasticsearch.yml

```diff
@@ -384,4 +384,7 @@ script.max_size_in_bytes: {{SCRIPT_MAX_SIZE_IN_BYTES}}
 {{#SCRIPT_PAINLESS_REGEX_ENABLED}}
 script.painless.regex.enabled: {{SCRIPT_PAINLESS_REGEX_ENABLED}}
 {{/SCRIPT_PAINLESS_REGEX_ENABLED}}
+{{#BOOTSTRAP_MEMORY_LOCK}}
+bootstrap.memory_lock: {{BOOTSTRAP_MEMORY_LOCK}}
+{{/BOOTSTRAP_MEMORY_LOCK}}
```

Comment:
Note that `bootstrap.memory_lock` is already specified at the top of the this file.

Reply:
thanks!